diff --git a/config.def.h b/config.def.h
index 5d32902..74488b6 100644
--- a/config.def.h
+++ b/config.def.h
@@ -177,6 +177,9 @@ static MouseShortcut mshortcuts[] = {
 #define MODKEY Mod1Mask
 #define TERMMOD (ControlMask|ShiftMask)
 
+static char *openurlcmd[] = { "/bin/sh", "-c", "~/Projects/dotfiles/st/linkgrabber.sh", "externalpipe", NULL };
+static char *openscreeneditor[] = { "/bin/sh", "-c", "~/Projects/dotfiles/st/editscreen.sh", "externalpipe", NULL };
+
 static Shortcut shortcuts[] = {
 	/* mask                 keysym          function        argument */
 	{ XK_ANY_MOD,           XK_Break,       sendbreak,      {.i =  0} },
@@ -192,6 +195,8 @@ static Shortcut shortcuts[] = {
 	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
 	{ TERMMOD,              XK_I,           iso14755,       {.i =  0} },
 	{ XK_ANY_MOD,           XK_F6,          swapcolors,     {.i =  0} },
+	{ TERMMOD,				XK_U,			externalpipe,	{.v = openurlcmd} },
+	{ TERMMOD,				XK_E,			externalpipe,	{.v = openscreeneditor} },
 };
 
 /*
